google/cloud/forseti/notifier/notifier.py

```diff
@@ -49,9 +49,8 @@ def find_notifiers(notifier_name):
         for filename in dir(module):
             obj = getattr(module, filename)
 
-            if inspect.isclass(obj) \
-                    and issubclass(obj, BaseNotification) \
-                    and obj is not BaseNotification:
+            if inspect.isclass(obj) and issubclass(obj, BaseNotification) and (
```

Comment:
@red2k18 this change should probably not be in this PR since it's not related to instrumentation

Reply:
Removed